src/Cmd/New.cc

```diff
@@ -23,54 +25,62 @@ const Subcmd NEW_CMD = //
         .setDesc("Create a new cabin project")
         .addOpt(OPT_BIN)
         .addOpt(OPT_LIB)
+        .addOpt(OPT_MODULES)
         .setArg(Arg{ "name" })
         .setMainFn(newMain);
 
 static constexpr std::string_view MAIN_CC =
     "#include <iostream>\n\n"
     "int main() {\n"
-    "  std::cout << \"Hello, world!\" << std::endl;\n"
-    "  return 0;\n"
+    "    std::cout << \"Hello, world!\" << std::endl;\n"
+    "    return 0;\n"
+    "}\n";
+
+static constexpr std::string_view MAIN_MODULES_CC =
+    "import std;\n\n"
+    "int main() {\n"
+    "    std::println(\"Hello, world!\");\n"
+    "    return 0;\n"
     "}\n";
 
 static std::string getAuthor() noexcept {
   try {
     git2::Config config = git2::Config();
     config.openDefault();
-    return config.getString("user.name") + " <" + config.getString("user.email")
-           + ">";
+    return fmt::format("{} <{}>", config.getString("user.name"),
+                       config.getString("user.email"));
   } catch (const git2::Exception& e) {
     spdlog::debug("{}", e.what());
     return "";
   }
 }
 
-std::string createCabinToml(const std::string_view projectName) noexcept {
-  std::string cabinToml = "[package]\n"
-                          "name = \"";
-  cabinToml += projectName;
-  cabinToml += "\"\n"
-               "version = \"0.1.0\"\n"
-               "authors = [\"";
-  cabinToml += getAuthor();
-  cabinToml += "\"]\n"
-               "edition = \"20\"\n";
-  return cabinToml;
+std::string createCabinToml(const std::string_view projectName,
+                            const bool useModules) noexcept {
+  return fmt::format("[package]\n"
```

Comment:
please make another PR for the use of fmt::format. i can merge it independently.